SeaPublicWebsite/Views/StaticPages/PrivacyPolicy.cshtml

```diff
@@ -0,0 +1,184 @@
+@using SeaPublicWebsite.Controllers
+@{
+    ViewBag.Title = $"Privacy Notice on {Constants.SERVICE_NAME}";
```

Comment:
I thought the capitatalisation on titles was sentence case (Privacy notice... rather than Privacy Notice...) but I could be wrong

Reply:
Good catch